.circleci/config/@config.yml

```diff
@@ -99,10 +99,10 @@ workflows:
           requires:
             - tag_and_push
           <<: *staging_branch_only_filter
-      - deploy_to_sysdig_integration_cluster:
-          requires:
-            - tag_and_push
-          <<: *staging_branch_only_filter
+      # - deploy_to_sysdig_integration_cluster:
```

Comment:
If this remains commented out and not removed, can you please add a comment with like a TODO to know what's needed to later remove this?